### DISCUSSION TITLE
Verständnisfrage mehrere Wechselrichter, ein Akku System --> max Leistung ins Netz begrenzen

### DISCUSSION THREAD

**Discussion by futzicomp** [hoylabs/OpenDTU-OnBattery#607] [Q&A]
Sorry, dass ich hier auf Deutsch schreibe, aber ich tu mich damit einfach leichter....

Folgende Voraussetzungen liegen bei mir vor:
1) 2x 405W Module --> Victron --> Pylontech US2000C --> HM-600 vorrangig für Deckung Grundbedarf in der Nacht 
2) 4x 435W Module --> HM1500 Geplant ist möglichst alles Selbst zu verbrauchen  (Wärmepume / E-Auto / evt. Klimaanlage)
3) 2x 405W Module--> HM600 Gebrauch wie Nr. 2
4) OpenDTUo.B. und Homeassistant laufen. Bei (1) ist Solarpasstrough eingestellt und Leistung begrenzt
Ich möchte gesetzeskonform die 600W/800W max Netzeinspeisung nicht überschreiten, jedoch einen möglichst hohen Eigenverbrauch realiseren können.( Deshalb die ganze Armada ;-) )

Wie und was stelle ich bei openDTU o.B. dazu am besten ein? ( oder was kann ich tun um die Netzeinspeisung aller Wechselrichter auf insgesamt 600/800W zu begrenzen UND trotzdem möglichst viel Eigenverbrauch meiner Solarernte zu nutzen)
 
Verzeiht mir wenn ich die Antwort auf mein Problem hier hätte finden müssen. Ich habe wirklich lange gesucht.  Aber momentan stehe ich auf dem Schlauch, wie ich das mit den derzeitigen Mitteln realisieren könnte. Auf euren Input bin ich gespannt. Vielen Dank im Voraus dafür ;-)

**Comment by spcqike** [comment]
> Ich möchte gesetzeskonform die 600W/800W max Netzeinspeisung nicht überschreiten

dann erschließt sich mir dein Setup nicht. Dafür reicht 1 Wechselrichter und alle Paneele an Solarladeregeln und die entsprechend große Batterie.

mit 600W kann man auch 14.4kWh am Tag einspeisen. 

dein HM-1500 bspw. dürftest du nur "gesetzeskonform" mit 600W einspeisen lassen, wenn alle anderen aus sind / nicht einspeisen. und wie man das bei der Anmeldung angibt, keine Ahnung.


zu deiner Frage:
am einfachsten wäre es, "einfach" alle Paneele auf eine Batterie und diese an einen großen Wechselrichter (HMS-2000 oder HMT-2250). das bedarf aber entsprechende Hardware.

mit der openDTU-onBattery ist es aktuell nicht möglich, unterschiedliche WEchselrichter zu begrenzen, geschweige denn unterschiedliche Quellen (mit Batterie und ohne) zu berücksichtigen.

Es gibt wohl diverse Skripte online, die mehrere Hoymiles über die openDTU steuern können, wie die aber mit unterschiedlichen Quellen umgehen, und wie die dafür sorgen, dass die Batterie nicht zu tief entladen wird, weiß ich auch nicht.


du könntest sowas hier versuchen:
1. Einspeisung nur bei Nacht, tagsünber geht alles in die Batterie
2.  geregelt über openDTU-oB mit Nulleinspeisungsziel
3. voll offen, ohne Regelung (oder hat Nr.3 auch ne Batterie?)

>  oder was kann ich tun um die Netzeinspeisung aller Wechselrichter auf insgesamt 600/800W zu begrenzen UND trotzdem möglichst viel Eigenverbrauch meiner Solarernte zu nutzen

wie gesagt, soweit mir bekannt, geht es bei den 600W Balkonkraftwerk nicht um die EInspeisung ins öffentliche Netz, sondern um die Erzeugung allgemein. du kannst keine 2000W erzeugen und direkt verbrauchen und sagen "0 EInspeisung, es ist nur ein Balkonkraftwerk da nicht > 600W ins öffentliche Netz eingespeist wird".

**Reply by futzicomp** [reply]
Vielen Dank für die Zeit, die du für mich mit deiner ausführlichen Antwort genommen hast!!!

Ohne jetzt eine riesen Diskussion aufzumachen: Meine bisherige Auffassung war: Ich kann (Stand 2023) >2000Wp installieren, solange ich nicht mehr als 600W ins NETZ(!) einspeise. Meine bisherige Denkweise war, das "Netz" erst nach dem Stromzähler beginnt, oder liege ich da fatalerweise falsch?
Hat jemand fundierte Kenntnisse?

BtW. Einen Sinn in der Begrenzung der WR Leistung sehe ich nach genauerem Überlegen nun schon: Wenn ich > 2kW über eine Schuko in mein Hausnetz schicke könnte die Phase/ Steckdose überlastet werden ?

**Reply by spcqike** [reply]
Wenn ich es richtig verstanden habe, gibt es, Stand 2023, keine Begrenzung der modulleistung. Der wechselrichter hingegen darf nicht mehr als 600W einspeisen können. Einspeisen ins hausnetz. Damit ist es ein kleines balkonkraftwerk. 

Für 2024 war angefacht die modulleistung auf … 2000Wp? Oder so zu begrenzen. Dafür die wechselrichter auf 800W zu erhöhen. Gleichzeitig Duldung der Ferraris Zähler und SchuKo. (Für 2023 gibt es die Freigabe noch nicht. Der Netzbetreiber oder Vermieter kann eine einspeisesteckdose vorschreiben) 

die 600/800 Watt kommen wohl scheinbar daher, dass Laien steckerfertige Anlagen einfach in sicherungskreise einstecken können sollen, ohne separate Kreise nur für PV vorzusehen. 

Eine Leitung hat heute normalerweise 1.5qmm und einen 16A LS. Mit den 2.6A (600W), die man produzieren kann, kommt man auf dem sicherungskreis auf 4.2kW. Nicht schön, aber verträglich. 

Mit einem 2kW wechselrichter kann man auf dem sicherungskreis halt 5.6kW (24.7A) verbrauchen, ohne dass der LS auslöst. Das ist weder für die Steckdosen, noch für die Geräte gut, sollten diese einen Fehler haben. 

Die gesamte inverter Leistung bestimmt die „Größe“ der Anlage. 2x300W ist heute noch ne balkonanlage. 1x800W mit Zertifikat der drosselung auch. 2x600W oder mehr, ohne Zertifikate, nicht. 

Wie viel davon „exportiert“ wird, ist egal. Es zählt die inverter Leistung. 

Im Gegenteil, bei größeren Anlagen gibt (gab?) es ja noch die 70% Regel. Es muss(te) sicher gestellt werden, dass nicht mehr als 70% der Leistung eingespeist (ins öffentliche Netz) wird. Das fällt bei Balkonkraftwerken, aufgrund der geringen Leistungen, weg. 

So zumindest mein Kenntnisstand :)
